<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="polaczPrzycisk">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>620</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Połącz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wyjdzPrzycisk">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>620</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyjdź</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>620</y>
      <width>681</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>620</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Informacje od robota:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dane wysyłane:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="testBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>150</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="rs232all">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>270</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>590</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Naciśnij &quot;ESC&quot;, aby przerwać połączenie</string>
    </property>
   </widget>
   <widget class="QLabel" name="cameraView">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>12</y>
      <width>761</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="oknoRozmowy">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>30</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="przodLabel">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>410</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Jazda do przodu</string>
    </property>
   </widget>
   <widget class="QLabel" name="lewoLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>430</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Jazda w lewo</string>
    </property>
   </widget>
   <widget class="QLabel" name="prawoLabel">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>430</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Jazda w prawo</string>
    </property>
   </widget>
   <widget class="QLabel" name="tylLabel">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>450</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Jazda do tyłu</string>
    </property>
   </widget>
   <widget class="QLabel" name="kameraGoraLabel">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>410</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kamera do góry</string>
    </property>
   </widget>
   <widget class="QLabel" name="kameraDolLabel">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>440</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kamera w dół</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
